### PR TITLE
Style fix for insertion in MVC dialog

### PR DESCRIPTION
**Pull request by rgoessl**

### CODE REVIEW

File: d2l-unlink-outcomes-picker.js [hunk 1]
```diff
@@ -336,11 +350,11 @@ class UnlinkOutcomesPicker extends LocalizedLitElement {
 
 		this.dispatchEvent(
 			new CustomEvent(
-				'd2l-unlink-outcomes-picker-confirm',
+				'd2l-unlink-outcomes-picker-import',
 				{
 					bubbles: false,
 					detail: {
-						unlinkAction: () => Valence.bulkUnlinkOutcomes( this.registryId, toUnlink ),
+						unlink: toUnlink,
```
Comment (rgoessl) [inline d2l-unlink-outcomes-picker.js:357]: Functions can't get passed to FRA through MVC dialog

File: internal/valence.js [hunk 2]
```diff
@@ -11,14 +11,6 @@ export default {
 			host += '/';
 		}
 	},
-    
-	bulkUnlinkOutcomes: function( registryId, outcomeIds ) {
```
Comment (rgoessl) [inline internal/valence.js:15]: Call moved to FRA

File: d2l-unlink-outcomes-picker.js (hunk 1 above)
Comment (mpharoah-d2l) [reply d2l-unlink-outcomes-picker.js:357]: Does the FRA actually need to get the function, though? Couldn't the LMS handle it?

Comment (rgoessl) [reply d2l-unlink-outcomes-picker.js:357]: FRA has a shared confirmation dialog for this picker case as well as single outcome unlink. So action must be invoked by FRA

**Pull request merged**